docs/user/reference/metrics/uuid.md

```diff
@@ -100,6 +100,7 @@ Glean.user.clientId.generateAndSet();
 ### `set`
 
 Sets a UUID metric to a specific value.
+Accepts any UUID version.
```

Comment:
We accept more than any UUID version... But I agree that we should not advertise that too much :)

Reply:
well, technically anything not RFC'd is still _a version_